tedana/tests/test_stats_getfbounds.py

```diff
@@ -0,0 +1,15 @@
+"""
+Tests for tedana.stats.getfbounds
+"""
+
+import numpy as np
+import pytest
+
+from tedana.stats import getfbounds
+
+
+def test_getfbounds():
+    good_inputs = range(1, 12)
```

Comment:
In the absence of a comment or named number, it's not clear why the range is 1,12, as another example of a mysterious number.

Reply:
I believe that those values correspond to reasonable numbers of echoes. I can add a comment to clarify.

Reply:
That would be fantastic, thank you!

Reply:
That said, I don't think it has to get fixed before merging this because that was already hardcoded; you haven't actually changed the test itself. Sorry if that wasn't clear!